doc/specs/vulkan/chapters/fundamentals.txt

```diff
@@ -268,10 +268,12 @@ frees.
 It is an application's responsibility to track the lifetime of Vulkan
 objects, and not to destroy them while they are still in use.
 
-Application-owned memory is immediately consumed by any Vulkan command it is
-passed into.
-The application can: alter or free this memory as soon as the commands that
-consume it have returned.
+[[fundamentals-objectmodel-lifetime-acquire]]
+The ownership of an application-owned memory is immediately acquired by any
```

Comment:
"an application-owned memory" seems like it should be just "application-owned memory". The "an" is extraneous.